validate_1.0.0-rc.2/checksum.py

```diff
@@ -0,0 +1,50 @@
+# !/usr/bin/env python3
+# -*- coding: utf-8 -*-
+ 
+ 
+import hashlib
+import os
+ 
+def get_checksum(filename, hash_function):
+    """Generate checksum for file baed on hash function (MD5 or SHA256).
```

Comment:
All these comments seem unnecessary, considering you have hardcoded algorithm and input in the code below. Can be removed

Reply:
this has been removed